aea/crypto/cosmos.py

```diff
@@ -639,14 +639,21 @@ def _get_init_transaction(
         :param chain_id: the Chain ID of the CosmWasm transaction. Default is 1 (i.e. mainnet).
         :return: the unsigned CosmWasm InitMsg
         """
+
+        if amount == 0:
+            init_funds = []
+        else:
+            init_funds = [{"amount": str(amount), "denom": denom}]
```

Comment:
This is fix for 

{
  "height": "0",
  "txhash": "629A46E37BD278A21459BBACAD8CD3F7649259F339EE55DF2F52EE9A6270C106",
  "codespace": "sdk",
  "code": 10,
  "raw_log": "invalid coins"
}

Apparently sending 0 coins is not an option anymore